dig_core/lib/src/domain/entity/staking/market.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'market.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class Market extends Equatable {
```

Comment:
I often use this tool [quicktype](https://quicktype.io) for huge data classes. You know any tools that support better for dart ?

Reply:
> I often use this tool [quicktype](https://quicktype.io) for huge data classes. You know any tools that support better for dart ?

I'm using only that tool:
https://javiercbk.github.io/json_to_dart/